robolectric/src/test/java/org/robolectric/RobolectricTestRunnerTest.java

```diff
@@ -103,14 +103,14 @@ public void shouldLoadDefaultsFromPropertiesFile() throws Exception {
             "reportSdk: 234\n" +
             "shadows: org.robolectric.shadows.ShadowView, org.robolectric.shadows.ShadowViewGroup\n" +
             "application: org.robolectric.TestFakeApp");
-    assertConfig(configFor(Test2.class, "withoutAnnotation", properties),
-        432, "--none", "from-properties-file", "from/properties/file/res", "from/properties/file/assets", 234, new Class[] {ShadowView.class, ShadowViewGroup.class});
+    assertConfig(configFor(Test7.class, "withoutAnnotation", properties),
+        432, "--none", "from-properties-file", "from/properties/file/res", "from/properties/file/assets", 234, new Class[] {ShadowView.class, ShadowViewGroup.class}, BuildConfigConstants3.class);
   }
 
   @Test
   public void withEmptyShadowList_shouldLoadDefaultsFromPropertiesFile() throws Exception {
     Properties properties = properties("shadows:");
-    assertConfig(configFor(Test2.class, "withoutAnnotation", properties), -1, "--default", "", "res", "assets", -1, new Class[] {});
+    assertConfig(configFor(Test7.class, "withoutAnnotation", properties), -1, "--default", "", "res", "assets", -1, new Class[] {}, BuildConfigConstants3.class);
```

Comment:
The `Test7` seems wrong and not pertinent here. These methods are for testing property loading, not overriding. Furthermore, this change in fact causes the issue in this pull request to no longer be covered by this test, as the issue occurred when the overlay config did _not_ define the constant properties. I propose removing the `Test7` and reverting to `Test2` in testing properties loading.